### PR TITLE
Fundamentals 5: Update to follow layout style

### PR DESCRIPTION
**Pull request by anasmustafa123**
## This PR
applying the required changes

## Issue
Related to [27119](https://github.com/TheOdinProject/curriculum/issues/27119)

## Pull Request Requirements

- [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
 The title of this PR follows the `location of change: brief description of change format`, e.g. Intro to HTML and CSS lesson: `Fix link text`
- [x] The Because section summarizes the reason for this PR
- [x] The `first` section has a bullet point list describing the changes in this PR
- [x] If this PR addresses an open issue, it is linked in the Issue section
- [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
- [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by anasmustafa123** [comment]
Sorry i saw that there was some conflict so reverted the changes then i ran 
```
git pull origin main
git fetch origin 
git checkout first
git merge main
```
to update my local branch
then while pushing the code i might neglected some conflicts

File: foundations/javascript_basics/object_basics.md [hunk 1]
```diff
@@ -176,6 +176,10 @@ If you feel yourself getting overwhelmed or stuck, don't be afraid to go back an
 
 The following questions are an opportunity to reflect on key topics in this lesson. If you can't answer a question, click on it to review the material, but keep in mind you are not expected to memorize or master this knowledge.
 
+- [What is the difference between objects and arrays?](https://javascript.info/object#summary)
+- [How do you access object properties?](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Basics#bracket_notation)
+- [What is `Array.prototype.map()` useful for?](https://www.youtube.com/watch?v=HB1ZC7czKRs&t=233s)
+- [What is `Array.prototype.reduce()` useful for?](https://youtu.be/HB1ZC7czKRs?t=467)
```
Comment (thatblindgeye) [inline foundations/javascript_basics/object_basics.md:182]: These are duplicated items and should be removed

File: javascript/organizing_your_javascript_code/json.md [hunk 2]
```diff
@@ -4,16 +4,32 @@ JSON (JavaScript Object Notation) is a standardized format for structuring data.
 
 Fortunately, there isn't much to learn here. We're only including a lesson on it because some formatting rules can cause confusion if you aren't aware of them. Spend 10-15 minutes going through the following resources and you'll be good to go.
 
+### Lesson overview
```
Comment (thatblindgeye) [inline javascript/organizing_your_javascript_code/json.md:7]: This file shouldn't have been updated. Can you revert these changes?

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

File: foundations/javascript_basics/object_basics.md (hunk 1 above)
Comment (anasmustafa123) [reply foundations/javascript_basics/object_basics.md]: done, sorry for the mistake

File: javascript/organizing_your_javascript_code/json.md (hunk 2 above)
Comment (thatblindgeye) [reply javascript/organizing_your_javascript_code/json.md:7]: @anasmustafa123 just checking that you've seen this comment as well, I see you resolved the above comment about duplicated content.

Comment (anasmustafa123) [reply javascript/organizing_your_javascript_code/json.md]: @thatblindgeye  done, sorry again

**Review by thatblindgeye — APPROVED** [review]
